packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue

```diff
@@ -171,15 +171,34 @@ const rootBoundary = computed<RootBoundary | undefined>(() =>
   props.boundary === 'document' || props.boundary === 'viewport' ? props.boundary : undefined
 )
 
-onKeyStroke(
-  'Escape',
-  () => {
-    modelValue.value = !modelValueBoolean
-  },
-  {target: splitButton}
-)
-
 const referencePlacement = toRef(() => (!splitBoolean.value ? splitButton.value : button.value))
+
+onKeyStroke('Escape', () => (modelValue.value = !modelValueBoolean), {target: referencePlacement})
+onKeyStroke('Escape', () => (modelValue.value = !modelValueBoolean), {target: floating})
```

Comment:
```suggestion
onKeyStroke('Escape', () => {modelValue.value = !modelValueBoolean}, {target: referencePlacement})
onKeyStroke('Escape', () => {modelValue.value = !modelValueBoolean}, {target: floating})
```
Whether or not it matters in this context, 

When you do 
```ts
let a = 1
const b = () => (a = 2)
```
The return type of b is not `void`, the return type is `number`. So, I prefer to never use that syntax due to unexpected complications

Reply:
I think that is some eslint or something completed it like that. Will change.